### PR TITLE
Bring over about page

### PR DESCRIPTION
**Pull request by nlitwin**
## Description

Bringing over the `/about` page and the `.j-link` class for styling links

## Sub-Tasks

- [x] I have added this PR to the Journaly Kanban project ✅

## Screenshots

journaly_about_page

### CODE REVIEW

**Comment by nlitwin** [comment]
Sounds good! I'll do that

File: frontend/components/Site/Home/Home.tsx [hunk 1]
```diff
@@ -11,7 +11,7 @@ const Home = () => {
       <VideoSection />
       <TestimonialsSection />
     </div>
-  )
-}
+  );
+};
 
-export default Home
```
Comment (robin-macpherson) [inline frontend/components/Site/Home/Home.tsx]: All these added semi-colons and double-quotes! 😬

File: frontend/components/Site/Home/HomeSection.tsx [hunk 2]
```diff
@@ -38,7 +38,7 @@ const HomeSection = ({ sectionHeading, grey = false, children }) => {
         }
       `}</style>
     </section>
-  )
-}
+  );
+};
 
-export default HomeSection
```
Comment (robin-macpherson) [inline frontend/components/Site/Home/HomeSection.tsx]: Same here with the semis and quotes

File: frontend/components/Site/Home/ReasonsSection.tsx [hunk 3]
```diff
@@ -84,6 +80,6 @@ const ReasonCard = ({ image, reasonNumber }) => (
       }
     `}</style>
   </div>
-)
+);
 
-export default ReasonsSection
```
Comment (robin-macpherson) [inline frontend/components/Site/Home/ReasonsSection.tsx]: Same here with the semis and quotes

**Comment by robin-macpherson** [comment]
Started leaving some comments on the files but since it's basically every file I guess I general comment is better lol. Looks like semi-colons and double quotes got added to basically every file by accident.

**Comment by nlitwin** [comment]
@robin-macpherson Ya not sure what happened here... I merged master into the `about-page` branch, fixed the one line conflict and pushed it up like normal. After I merged it in, I realized I was looking at the cached version of the Files tab and saw that there were all these linting changes on the files. I was planning on creating a PR to fix all the linting changes

**Comment by robin-macpherson** [comment]
@nlitwin Not a huge deal! 😊 I think we can just do it as part of the next feature branch, WDYT?

**Pull request merged**